Mathlib/Data/Int/Init.lean

```diff
@@ -282,6 +282,46 @@ lemma strongRec_of_lt (hn : n < m) : m.strongRec lt ge n = lt n hn := dif_pos _
 
 end strongRec
 
+/-! ### mul -/
+
+-- We want to use these lemmas earlier than the lemmas simp can prove them with
+
+@[simp, nolint simpNF] protected lemma mul_le_mul_left (ha : 0 < a) : a * b ≤ a * c ↔ b ≤ c where
```

Comment:
@YaelDillies I think these should probably be marked `simp low` instead of `nolint simpNF`. Do you agree?

Reply:
Do you mean `simp high`? Otherwise tjey wouldn't be in simp normal form.

And, yes, sure, tweaking the priority is another option